convention.md

```diff
@@ -475,3 +484,9 @@ homie/$broadcast/alert ← "Intruder detected"
 ```
 
 Any other topic is not part of the Homie convention.
+
+### Optional Extensions
+
+In addition to homie-core the device may support multiple extensions which are defined in separate conventions.
+Every extension is identified by a unique id, which is similar to a topic id but also allows dots.
```

Comment:
"which is similar to a topic id but also allows dots" is superfluous, because in the next sentence you clarify/narrow down the allowed id to a reverse domain name. Maybe combine those two sentences?